site-src/geps/pending-geps.md

```diff
@@ -0,0 +1,58 @@
+#### Gateway API Enhancment Proposal Status November 2023
+
+This table lists the Gateway API Enhancement Proposals that are currently in Provisional (P) or
+Experimental (E) status (see row 2, Status).
+
+The results of compliance testing for each GEP is listed in row 3 (Compliance).  The Compliance
+codes are:
+
+* NS - Compliance tests not started
+* NF - Compliance tests not finished (being developed)
+* R  - Compliance tests ready for implementors to run
+
+This table lists November 2023 implmentors of beta, alpha, or GA status.  Each implementor can mark
+this table with a code for their support and/or planned support for each of the proposed enhancements,
+using these codes:
+
+* blank - Not planned
+* PP - Planned, in progress
+* RR - Ready in next release
+* C  - Complete in current release
+
+For example, an imaginary implementor called Imaginary has support completed for GRPC Route (GEP-1016), planned
+ and in progress for HTTPRoute Timeouts (GEP-1742), but otherwise not planned for any other GEP.
+Imaginary's row in the table would look like this:
+
+
+|      | GEP-1619: Session Persistence | GEP-1867: Per-Gateway Infrastructure | GEP-957: Destination Port Matching | GEP-1016: GRPCRoute | GEP-1324: Service Mesh in Gateway API | GEP-1426: xRoutes Mesh Binding | GEP-1651: Gateway Routability | GEP-1686: Mesh conformance testing plan | GEP-1709: Conformance Profiles | GEP-1742: HTTPRoute Timeouts | GEP-1748: Gateway API Interaction with Multi-Cluster Services | GEP-1897: BackendTLSPolicy | GEP-1911: Backend Protocol Selection | GEP-2162: Supported features in GatewayClass Status | GEP-2257: Gateway API Duration Format |
```

Comment:
I don't think it's worth including provisional or unimplementable GEPs in the list (That would exclude persistence, duration format, conformance profiles, and routability).

Many of the remaining GEPs have corresponding supported features, so we could get data from conformance reports (https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports) to help populate them.

Reply:
I imagined using this to gauge interest and planning for the provisional/experimental GEPs.  Only the first two are marked as provisional in the GEPS.  I think only the conformance profiles is unimplementable, but still it warrants interest and participation.   Which others are unimplementable?

Also, i was hoping to get group participation here, only saw a handfu (5) conformance reports.